src/snowflake/snowpark/mock/_connection.py

```diff
@@ -800,6 +800,9 @@ def create_coprocessor(self):
         # It's not necessary to mock this call.
         pass
 
+    def phase1_enabled(self):
```

Comment:
nit: can we add a comment that mock conneciton is not relevant here? Therefore always return False?